src/main/resources/static/error.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
you have added the error.html file in the static folder. is that correct?

Reply:
yes